autopush/settings.py

```diff
@@ -227,6 +256,15 @@ def update_rotating_tables(self):
 
         """
         today = datetime.date.today()
+        tomorrow = self._tomorrow()
+        if ((tomorrow.month != today.month) and
+                sorted(self.message_tables.keys())[-1] !=
+                tomorrow.month):
+            next_month = get_rotating_message_table(
+                self._message_prefix, 0, tomorrow)
```

Comment:
Why not remove the date passing addition to get_rotating_message_table and just pass a delta of 1 in?